### ISSUE TITLE
Basic design (home page, banners, logos, etc.)

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/dlme#134]
Requirements:
- [ ] Replace the blacklight logo with a DLME logo
- [ ] determine if it is still necessary to link  to dlme.clir.org

**Comment by jacobthill** [comment]
Requirements:
- [ ] Replace the blacklight logo with a DLME logo

**Comment by ggeisler** [comment]
@jacobthill Has anything changed in terms of whether there should be any Stanford/SUL visibility on the DLME site? 

Currently there is no Stanford/SUL visibility; the footer contains info on CLIR and DLF but no mention of Stanford/SUL. Not saying there should be, I just want to double-check that there hasn't been any change to the branding requirements since the last time we worked on this.

As an example of another collaborative project, for the [Vatican Spotlight project](https://spotlight.vatlib.it/overview) I made the visual design match their branding, but in the footer we did include a mention of SUL:

<img width="999" alt="Screen Shot 2019-10-14 at 2 46 57 PM" src="https://user-images.githubusercontent.com/101482/66785237-83c67200-ee91-11e9-846f-792a39dca72b.png">

Again, I'm not suggesting we **should** do something similar, just want to make sure we're aware of any new or different branding requirements before we start updating the branding and visual design.

**Comment by ggeisler** [comment]
@jacobthill The requirement in this ticket of `Replace the blacklight logo with a DLME logo` assumes that a DLME logo exists. Does it?

The only thing I can find is what's in the screenshot below (from https://dlme.clir.org/), which simply styles the body font of the site in a large text size. So I'm wondering if there should be another ticket (e.g., "Create DLME logo") that's a prereq for this one, or if we just want to replace the blacklight logo with a text-based "DLME" that is styled with CSS to look decent?

<img width="203" alt="Screen Shot 2019-10-14 at 3 04 40 PM" src="https://user-images.githubusercontent.com/101482/66786059-f7697e80-ee93-11e9-8d0f-0c8b6219c6f0.png">

**Comment by ggeisler** [comment]
@caaster During retro it came up that DLF perhaps shouldn't even be included as part of the site branding. So in addition to my previous comments, clarity on these issues would help move this ticket forward:

- Which project partners should be visible in the branding (e.g., logo's) and how prominent should each be?
- For project partners that should be part of the site branding, do they have preferred logos we should use?
- Will project partners have any new or different About page content (see https://spotlight.dlmenetwork.org/library/about/about for what we currently have)?

**Comment by caaster** [comment]
Zoom call with CLIR (Wayne, Elizabeth) & Mark, Cathy & Gary on 15 Nov 2019 provided much of the info needed by Gary.

**Comment by anarchivist** [comment]
New tickets will be created for necessary design changes
